flutter/lib/src/utils/multi_view_helper.dart

```diff
@@ -0,0 +1,14 @@
+import 'dart:ui';
+import 'package:meta/meta.dart';
+
+@internal
+class MultiViewHelper {
+  static bool isMultiViewEnabled() {
+    final dynamic uncheckedImplicitView = PlatformDispatcher.instance;
```

Comment:
We should probably use this wrapper instead 

https://github.com/getsentry/sentry-dart/blob/main/flutter/lib/src/integrations/on_error_integration.dart#L113C1-L153C2

e.g
```dart
final wrapper = dispatchWrapper ?? PlatformDispatcherWrapper(binding.platformDispatcher);
```

Reply:
maybe also makes sense then to move the platform dispatch wrapper to its own file